th/2012/spring.json

```diff
@@ -0,0 +1,777 @@
+			"detail": [{
+				"name": "BlackHole-SS"
+			}]
+		}, {
+			"status": "release",
+			"detail": [{
+				"name": "EmoCoNisT-FS"
+			}]
+		}, {
+			"status": "release",
+			"detail": [{
+				"name": "Genshokyou-fs"
+			}]
+		}, {
+			"status": "release",
+			"detail": [{
+				"name": "Google-Sub"
+			}]
+		}, {
+			"status": "release",
+			"detail": [{
+				"name": "Kami-Tama FS"
+			}]
+		}, {
+			"status": "release",
+			"detail": [{
+				"name": "Mame Fansub Group"
+			}]
+		}, {
+			"status": "release",
+			"detail": [{
+				"name": "RA-FS"
+			}]
+		}, {
+			"status": "release",
+			"detail": [{
+				"name": "Sunshine"
+			}]
+		}, {
+			"status": "release",
+			"detail": [{
+				"name": "ปคคปด-fs"
+			}]
+		}]
+	}, {
+		"name": "Acchi Kocchi",
+		"image": "https://myanimelist.cdn-dena.com/images/anime/5/46489.jpg",
+		"group": [{
+			"status": "release",
+			"detail": [{
+				"name": "ACE-Z Jr"
+			}]
+		}, {
+			"status": "release",
+			"detail": [{
+				"name": "CT-FS"
+			}, {
+				"name": "NERD-FS"
+			}]
+		}, {
+			"status": "release",
+			"detail": [{
+				"name": "TH-Kagami Fansub"
+			}]
+		}, {
+			"status": "release",
+			"detail": [{
+				"name": "The Anime - FS"
+			}]
+		}, {
+			"status": "release",
+			"detail": [{
+				"name": "YukiKaze-FS"
+			}]
+		}]
+	}, {
+		"name": "AKB0048",
+		"image": "https://myanimelist.cdn-dena.com/images/anime/3/74157.jpg",
+		"group": [{
+			"status": "release",
+			"detail": [{
+				"name": "ACE-Z"
+			}]
+		}, {
+			"status": "release",
+			"detail": [{
+				"name": "Chuy"
+			}]
+		}, {
+			"status": "release",
+			"detail": [{
+				"name": "Darkwing-fs"
+			}]
+		}, {
+			"status": "release",
+			"detail": [{
+				"name": "NoGod-FS"
+			}]
+		}, {
+			"status": "release",
+			"detail": [{
+				"name": "Suzukaze-FS"
+			}]
+		}]
+	}, {
+		"name": "Eureka Seven AO",
+		"image": "https://myanimelist.cdn-dena.com/images/anime/11/36313.jpg",
+		"group": [{
+			"status": "release",
+			"detail": [{
+				"name": "Grider-Fs"
+			}]
+		}, {
+			"status": "release",
+			"detail": [{
+				"name": "NOTZ-FANSUB"
+			}]
+		}, {
+			"status": "release",
+			"detail": [{
+				"name": "pandoramon"
+			}]
+		}]
+	}, {
+		"name": "Fate/Zero 2nd Season",
+		"image": "https://myanimelist.cdn-dena.com/images/anime/8/41125.jpg",
+		"group": [{
+			"status": "release",
+			"detail": [{
+				"name": "Akiraz-FS"
+			}]
+		}, {
+			"status": "release",
+			"detail": [{
+				"name": "Rhythms-subs"
+			}]
+		}, {
+			"status": "release",
+			"detail": [{
+				"name": "Takz-FS"
+			}]
+		}]
+	}, {
+		"name": "Gon",
+		"image": "https://myanimelist.cdn-dena.com/images/anime/6/36931.jpg",
+		"group": [{
+			"status": "release",
+			"detail": [{
+				"name": "Nuclear Fusion"
+			}]
+		}]
+	}, {
+		"name": "Haiyore! Nyaruko-san",
+		"image": "https://myanimelist.cdn-dena.com/images/anime/6/49081.jpg",
+		"group": [{
+			"status": "release",
+			"detail": [{
+				"name": "ACE-Z"
+			}, {
+				"name": "MunashiiKaze-FS"
```

Comment:
MunashiiKaze-FS was co-sub with ExOtIc
and work on co-sub with ACE-Z on Haiyore! Nyaruko-san F but they have wrong anime tag on menome.in.th